docs/README.md

```diff
@@ -1,21 +1,10 @@
-+++
-draft = "True"
-+++
+# Grafana Agent Documentation
 
-<p align="center"><img src="assets/logo_and_name.png" alt="Grafana Agent logo"></p>
+This directory contains documentation for Grafana Agent. It is split into two
+parts:
 
-Grafana Agent is an telemetry collector for sending metrics, logs,
-and trace data to the opinionated Grafana observability stack. It works best
-with:
-
-* [Grafana Cloud](https://grafana.com/products/cloud/)
-* [Grafana Enterprise Stack](https://grafana.com/products/enterprise/)
-* OSS deployments of [Grafana Loki](https://grafana.com/oss/loki/), [Prometheus](https://prometheus.io/), [Cortex](https://cortexmetrics.io/), and [Grafana Tempo](https://grafana.com/oss/tempo/)
-
-
-- Grafana Agent uses less memory on average than Prometheus – by doing less
-  (only focusing on `remote_write`-related functionality).
-- Grafana Agent allows for deploying multiple instances of the Agent in a
-  cluster and only scraping metrics from targets that are running on the same host.
-  This allows for distributing memory requirements across the cluster
-  rather than pressurizing a single node.
+* `user/`: Documentation for users. This directory is hosted on
+  [grafana.com/docs/agent](https://grafana.com/docs/agent/latest/) and we
+  recommend interacting with it there instead of viewing the markdown on
+  Github.
+* `rfcs/`: RFCs for proposals relating to Grafana Agent.
```

Comment:
My gut is to just have `rfcs` in the normal user docs. https://prometheus.io/docs/introduction/design-doc/ has not led to major confusion, either.

Reply:
I think it'd be nice to host them, but I'm concerned that they might be subject to extra scrutiny if they're hosted on grafana.com. Keeping them separate seems the most expedient for now.